src/backend/distributed/deparser/qualify_type_stmt.c

```diff
@@ -0,0 +1,194 @@
+/*-------------------------------------------------------------------------
+ *
+ * qualify_type_stmt.c
+ *	  Functions specialized in fully qualifying all type statements. These
+ *	  functions are dispatched from qualify.c
+ *
+ *	  Fully qualifying type statements consists of adding the schema name
+ *	  to the subject of the types as well as any other branch of the
+ *	  parsetree.
+ *
+ *	  Goal would be that the deparser functions for these statements can
+ *	  serialize the statement without any external lookups.
+ *
+ * Copyright (c) 2019, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "access/heapam.h"
+#include "access/htup_details.h"
+#include "catalog/namespace.h"
+#include "catalog/objectaddress.h"
+#include "catalog/pg_type.h"
+#include "distributed/deparser.h"
+#include "distributed/metadata/namespace.h"
+#include "nodes/makefuncs.h"
+#include "parser/parse_type.h"
+#include "utils/lsyscache.h"
+
+static Oid
```

Comment:
in general, missing function comments in the file. And, why do we use Postgres function name conventions here (and some other places as well)?

I think especially these  kinds of utility functions should be named with Citus convention to cause less confusion. When I see a function name, it is mostly useful to understand whether that's a Citus or Postgres function.

Reply:
re missing docs, this function is the only one where we can add useful documentation (will add) the other would be either very repetitive or very trivial. This is compensated by having extensive documentation on this file and even more so on `qualify.c` as qualification of a tree is an important concept.

these functions are all in a form of QualifyNameOfStmtStruct where the documentation would be
```
/*
 * QualifyNameOfStmtStruct qualifies a NameOfStmtStruct
 */
```

By not having that on there the hope is people actually look at the top of the file to get a sense of what these functions are doing.